test/unit_test/character_sheet_unit_tests/SpeedAndDefenseSupportTests.java

```diff
@@ -54,6 +54,14 @@ public void SetLightPlateGetSpeed6() {
 	}
 
 	@Test
+	public void SetBackgroundToCityElfSetArmorHeavyLeatherGetSpeed11() {

```

Comment:
Itt nagyon sokat csinálsz, elég ha állítasz egy light mail-t vagy heavy mail-t vagy valamit és ugyanazt csinálod mint a feljebbi tesztben. Valami indokolja, hogy a backgroundot is átállítsd?

Reply:
Igen, mivel az is befolyásolja a speedet igy kell egy olyan teszt is ahol mind a három függőséget tesztelem egyben.